### ISSUE TITLE
@ character

### ISSUE THREAD

**Issue by pathcl** [leandrotoledo/python-telegram-bot#86]
If I type @ in a chat I got this. (using https://github.com/leandrotoledo/python-telegram-bot/blob/master/examples/echobot.py)

TypeError: b'hola @honguitobot' is not JSON serializable

**Comment by franciscod** [comment]
The error isn't about the `@`, it's about the byte array not being serializable. Hmmm... interesting

**Comment by franciscod** [comment]
I can reproduce on py3:
```
Traceback (most recent call last):
  File "echobot.py", line 66, in <module>
    main()
  File "echobot.py", line 44, in main
    echo(bot)
  File "echobot.py", line 59, in echo
    text=message)
  File "/home/fd/src/tgbots/venv/lib/python3.5/site-packages/telegram/bot.py", line 126, in decorator
    result = func(self, *args, **kwargs)
  File "/home/fd/src/tgbots/venv/lib/python3.5/site-packages/telegram/bot.py", line 158, in decorator
    result = request.post(url, data)
  File "/home/fd/src/tgbots/venv/lib/python3.5/site-packages/telegram/utils/request.py", line 88, in post
    data = json.dumps(data)
  File "/usr/lib64/python3.5/json/__init__.py", line 230, in dumps
    return _default_encoder.encode(obj)
  File "/usr/lib64/python3.5/json/encoder.py", line 199, in encode
    chunks = self.iterencode(o, _one_shot=True)
  File "/usr/lib64/python3.5/json/encoder.py", line 257, in iterencode
    return _iterencode(o, 0)
  File "/usr/lib64/python3.5/json/encoder.py", line 180, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: b'this is an echo test' is not JSON serializable
```

**Comment by franciscod** [comment]
Dropping the `.encode('utf-8')` part fixes it, and it works on 2.7.10 and 3.5.0

**Comment by franciscod** [comment]
also related to #72 ?
